odoo/tools/translate.py

```diff
@@ -292,6 +292,51 @@ def parse_xml(text):
 def serialize_xml(node):
     return etree.tostring(node, method='xml', encoding='unicode')
 
+
+MODIFIER_ATTRS = {"invisible", "readonly", "required", "column_invisible", "attrs", "states"}
+def xml_term_adapter(term_en):
+    """
+    Returns an `adapter(term)` funcion that will ensure the modifiers are copied
```

Comment:
May be clearify the requirements of `term_en` in the comment since it is a low level customization for `xml_translate`
The `term_en` used here and the `term` used for `adaper` should be valid terms for xml, so we remove the `try` and `expect`
Or we even ask `term_en` to be not pure text which contains xml part, so we can even remove the `ident`

Reply:
try/except were removed already, I'll check what cases lead to ident, but I think you are right that we can skip it by checking if term_en is text.